### DISCUSSION TITLE
Homebrew file error uninstalling

### DISCUSSION THREAD

**Discussion by but0004** [Homebrew/discussions#212] [Linux]
When i load my teminal this comes up:
`-bash:` /bin/brew: Is a directory
-bash: /bin/brew: Is a directory
-bash: /bin/brew: Is a directory
-bash: /home/but0004/.linuxbrew/bin/brew: No such file or directory `

i have done:

`but0004@penguin:~$` sudo /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/uninstall.sh)"
`/bin/bash:` line 193: brew: command not found
`Failed` to locate Homebrew!
`but0004@penguin:~$` curl -Lo /tmp/uninstall.sh https://raw.githubusercontent.com/Homebrew/install/master/uninstall.sh
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
`100`  9894  100  9894    0     0  23783      0 --:--:-- --:--:-- --:--:-- 23783
`but0004@penguin:~$` bash /tmp/uninstall.sh -p /home/but0004/.linuxbrew
`Warning:` Unrecognized option: '/home/but0004/.linuxbrew'
`Homebrew` Uninstaller
`Usage:` /tmp/uninstall.sh [options]
``    -p, --path=PATH  Sets Homebrew prefix. Defaults to /usr/local.
``        --skip-cache-and-logs
``                     Skips removal of HOMEBREW_CACHE and HOMEBREW_LOGS.
``    -f, --force      Uninstall without prompting.
``    -q, --quiet      Suppress all output.
``    -d, --dry-run    Simulate uninstall but don't remove anything.
``    -h, --help       Display this message.
`but0004@penguin:~$` sudo bash /tmp/uninstall.sh -p /home/but0004/.linuxbrew
`Warning:` Unrecognized option: '/home/but0004/.linuxbrew'
`Homebrew` Uninstaller
`Usage:` /tmp/uninstall.sh [options]
 ``   -p, --path=PATH  Sets Homebrew prefix. Defaults to /usr/local.
  ``      --skip-cache-and-logs
   ``                  Skips removal of HOMEBREW_CACHE and HOMEBREW_LOGS.
    -f, --force      Uninstall without prompting.
    -q, --quiet      Suppress all output.
    -d, --dry-run    Simulate uninstall but don't remove anything.
    -h, --help       Display this message.
`but0004@penguin:~$` bash /tmp/uninstall.sh --path=/home/but0004/.linuxbrew
`Failed` to locate Homebrew!
`but0004@penguin:~$` ^C
`but0004@penguin:~$` bash /tmp/uninstall.sh --path=/home/but0004/.linuxbrewbash /tmp/uninstall.sh --path=/home/but0004/.linuxbrew
`Warning:` Unrecognized option: '/tmp/uninstall.sh'
`Homebrew` Uninstaller
`Usage:` /tmp/uninstall.sh [options]
``    -p, --path=PATH  Sets Homebrew prefix. Defaults to /usr/local.
``        --skip-cache-and-logs
``                     Skips removal of HOMEBREW_CACHE and HOMEBREW_LOGS.
``    -f, --force      Uninstall without prompting.
``    -q, --quiet      Suppress all output.
``    -d, --dry-run    Simulate uninstall but don't remove anything.
``    -h, --help       Display this message.
`but0004@penguin:~$` 
how do i stop the messege?

**Comment by SMillerDev** [comment]
Something in your shell environment is trying to load homebrew. Check `~/.profile`, `~/.bash_profile` or `~/.bashrc` for references to brew.
